src/modules/translation/languages/en.json

```diff
@@ -761,5 +761,10 @@
     "rarity": "Rarity",
     "price": "Price",
     "beneficiary": "Beneficiary"
+  },
+  "edit_price_and_beneficiary_modal": {
+    "title": "Set price and address for {name}",
+    "price_label": "Price",
+    "beneficiary_label": "Beneficiary"
```

Comment:
These seem to be only in the english translations

Reply:
great catch!